js/map.js

```diff
@@ -357,7 +362,7 @@ function clearWindows() {
 //Option Window Code
 function ShowAdvanced(what) {
   if (what.innerHTML=='+Show Advanced') {
-    $('div#customAdvacned').show();	
+    $('div#customAdvacned').show();
```

Comment:
correct typo? `customAdvacned` to `customAdvanced`

Reply:
Just to build on this-- would have to fix here too: https://github.com/tomeshnet/tomesh.net/blob/map-createissue/map.html#L36